Tribler/Main/vwxGUI/home.py

```diff
@@ -90,37 +109,79 @@ def __init__(self, parent):
         hSizer.Add(StaticText(self, -1, " to see what others are sharing."))
 
         vSizer.Add(text, 0, wx.ALIGN_CENTER)
-        vSizer.AddSpacer(10)
-        vSizer.Add(scalingSizer, 0, wx.ALIGN_CENTER)
-        vSizer.AddSpacer(10)
+        vSizer.Add(scalingSizer, 1, wx.ALIGN_CENTER_HORIZONTAL | wx.ALIGN_TOP)
         vSizer.Add(hSizer, 0, wx.ALIGN_CENTER)
         vSizer.AddStretchSpacer()
 
+        # channel panel is for popular channel
+        self.channel_panel = ScrolledPanel(self, 1)
+        self.channel_panel.SetBackgroundColour(wx.WHITE)
+        self.channel_panel.SetForegroundColour(parent.GetForegroundColour())
+
+        v_chn_sizer = wx.BoxSizer(wx.VERTICAL)
+        v_chn_sizer.Add(
+            DetailHeader(self.channel_panel, "Select popular channels to mine"),
+            0, wx.EXPAND, 5)
+
+        self.loading_channel_txt = wx.StaticText(self.channel_panel, 1, 'Loading, please wait.')
+        v_chn_sizer.Add(self.loading_channel_txt, 1, wx.TOP | wx.ALIGN_CENTER_HORIZONTAL, 10)
+
+        self.chn_sizer = wx.FlexGridSizer(0, COLUMN_SIZE, 5, 5)
+
+        for i in range(0, COLUMN_SIZE):
+            if wx.MAJOR_VERSION > 2:
+                if self.chn_sizer.IsColGrowable(i):
+                    self.chn_sizer.AddGrowableCol(i, 1)
+            else:
+                self.chn_sizer.AddGrowableCol(i, 1)
+
+        v_chn_sizer.Add(self.chn_sizer, 0, wx.EXPAND, 5)
+
+        self.channel_panel.SetSizer(v_chn_sizer)
+        self.channel_panel.SetupScrolling()
+
+        vSizer.Add(self.channel_panel, 5, wx.EXPAND)
+
+        # video thumbnail panel
         self.aw_panel = ArtworkPanel(self)
         self.aw_panel.SetMinSize((-1, 275))
-        self.aw_panel.Hide()
+        # TODO(lipu): enable this when metadata PR is merged
```

Comment:
I guess the metadata PR is already merged. Have you tried to enable the next line?